### ISSUE TITLE
[Naxx] Gothik the Harvester encounter

### ISSUE THREAD

**Issue by peterdimitrov** [FeenixServerProject/Phoenix_1.12.1_Issue_tracker#5284]
At the phase after he comes is in the room, the doors don't open until he is completely dead.

Also mobs can fly through the gates, so that people on the living side must battle undead mobs and vice versa.

**Comment by Cabro** [comment]
Yes but you can cast through the doors, so you got that going for you. One cancels the other?
